api/v1alpha4/nutanixcluster_conversion.go

```diff
@@ -19,6 +19,7 @@ package v1alpha4
 import (
 	infrav1beta1 "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1beta1"
 	apiconversion "k8s.io/apimachinery/pkg/conversion"
+	//nolint:staticcheck
```

Comment:
had to add a nolint above every v1alpha4 import due to cyclic nature of generated code.

Reply:
Which check is failing here, maybe a comment would be helpful? (there is also a linter to force having comments we may want to enable)

Reply:
Added nolintlint and configured it to complain on lack of comments. It was complaining about the deprecation.